components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -12,23 +20,20 @@ import clsx from 'clsx';
 const MAX_SUBTITLE_ELEMENTS = 6;
 
 export type DividerType = 'full' | 'partial';
-export type InfoListItemProps = {
+export type InfoListItemProps = Omit<Omit<ListItemProps, 'title'>, 'divider'> & {
```

Comment:
I believe `Omit` will let you specify a list of properties to ignore to avoid this nesting

Reply:
I tried Googling for this but wasn't lucky enough. Can you specify how this is done? I tried `Omit<ListItemProps, ['title', 'divider']>` but obviously it was not the right way to do it.

Reply:
I stand corrected. I thought there was a way to do it, but it looks like there is not (I was probably thinking of the way `Exclude` works. 

We could write our own typescript helper function 😏

Reply:
😱